resources/public/api/conf/3.0/schemas/retrieve_property_bsas.json

```diff
@@ -1,154 +1,675 @@
+          "type": "string",
+          "example": "2021-22"
+        },
+        "summaryStatus": {
+          "type": "string",
+          "enum": [
+            "valid",
+            "invalid",
+            "superseded"
+          ],
+          "description": "The status of the this calculated summary, limited to the following possible values: \n\n valid \n\n invalid \n\n superseded"
+        }
+      },
+      "required": [
+        "calculationId",
+        "requestedDateTime",
+        "nino",
+        "taxYear",
+        "summaryStatus"
+      ]
     },
-    "bsas": {
+    "inputs": {
       "type": "object",
-      "description": "Object holding the BSAS for either a UK property non-Furnished Holiday Let (non-FHL) or for a UK Property Furnished Holiday Let (FHL) business income source.",
+      "description": "Object containing the inputs that were used in this calculation.",
       "properties": {
-        "total": {
-          "$ref": "definitions.json#/definitions/bsasTotal"
+        "businessId": {
+          "description": "An identifier for the business, unique to the customer. Must conform to the regular expression: ^X[A-Z0-9]{1}IS[0-9]{11}$",
+          "type": "string",
+          "example": "XAIS12345678910"
+        },
+        "typeOfBusiness": {
+          "type": "string",
+          "enum": [
+            "uk-property-fhl",
+            "uk-property-non-fhl"
+          ],
+          "description": "The type of business the calculation summary is for. Limited to the following possible values: \n\n uk-property-fhl \n\n uk-property-non-fhl"
+        },
+        "businessName": {
+          "description": "The name of the business.",
+          "type": "string",
+          "example": "My uk property business"
         },
-        "profit": {
-          "$ref": "definitions.json#/definitions/bsasProfit"
+        "accountingPeriodStartDate": {
+          "description": "The date the accounting period started.",
+          "type": "string",
+          "example": "2021-22"
         },
-        "loss": {
-          "$ref": "definitions.json#/definitions/bsasLoss"
+        "accountingPeriodEndDate": {
+          "description": "The date the accounting period ended.",
+          "type": "string",
+          "example": "2022-23"
         },
-        "incomeBreakdown": {
+        "source": {
+          "type": "string",
+          "enum": [
+            "MTD-SA"
+          ],
+          "description": "Specifies the source of the income data used in this calculation. Limited to the following possible values: \n\n MTD-SA"
+        },
+        "submissionPeriods": {
+          "type": "array",
+          "description": "Array containing the submission periods for the income source. A periodic summary will always have either a periodId or a submissionId, but not both.",
+          "items": {
+            "type": "object",
+            "properties": {
+              "periodId": {
```

Comment:
periodId requires the YYYY-MM-DD_YYYY-MM-DD to be put as a "pattern" : "YYYY-MM-DD_YYYY-MM-DD" and taken out the description